dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -78,6 +78,11 @@ $ oc set probe dc/nexus \
 	--failure-threshold 3 \
 	--initial-delay-seconds 30 \
 	-- echo ok
+oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://:8081/content/groups/public
```

Comment:
should they be? One is to test whether the container is live (which a simple cmd would test), vs the other one to test whether it's ready to serve content